src/test/java/ladder/UserTest.java

```diff
@@ -0,0 +1,27 @@
+package ladder;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
+    @Test
+    void 유저_이름은_Null이면_안된() {
+        Assertions.assertThatThrownBy(() -> {
+            User user = new User(null);
+        }).isInstanceOf(NullPointerException.class);
+    }
+
+    @Test
+    void 유저_이름은_빈값이면_안된() {
```

Comment:
```suggestion
    void 유저_이름은_Null이면_안된다() {
        Assertions.assertThatThrownBy(() -> {
            User user = new User(null);
        }).isInstanceOf(NullPointerException.class);
    }

    @Test
    void 유저_이름은_빈값이면_안된다() {
```

안된다에서 "다" 가 빠진 것 같아서 언급을 해봅니다.